### PR TITLE
Feature/malus bar

### PR DESCRIPTION
**Pull request by SeanPed**

### CODE REVIEW

File: src/app/components/DeleteButton/DeleteButton.module.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+button {
+  color: snow;
```
Comment (derjubu) [inline src/app/components/DeleteButton/DeleteButton.module.css:2]: Hex-codes (or similar) would be better

File: src/app/components/DeleteButton/DeleteButton.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './DeleteButton.module.css';
+
+export default function deleteButton(): JSX.Element {
+  return <button>X</button>;
```
Comment (derjubu) [inline src/app/components/DeleteButton/DeleteButton.tsx:5]: You should use an aria attribute, if there is no 'real text' on the button. Also, you could thing about using an SVG in this case.

**Review by derjubu — APPROVED** [review]
Lgtm 👍

File: src/app/components/DeleteButton/DeleteButton.module.css (hunk 1 above)
Comment (F-Kirchhoff) [reply src/app/components/DeleteButton/DeleteButton.module.css:2]: or use css variables

File: src/app/components/DeleteButton/DeleteButton.tsx (hunk 2 above)
Comment (F-Kirchhoff) [reply src/app/components/DeleteButton/DeleteButton.tsx:5]: maybe think about a button class where the text / svg inside the button is given to the component via a prop. in this way you dont have to write a new component for each button

File: src/app/components/Form/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+export default function InputForm(): JSX.Element {
+  return (
+    <form action="">
+      <label className={classes.labelTitle}>
+        Name des Gegenstandes
+        <br />
+        <input type="text" autoComplete="off" />
+      </label>
+      <label>
```
Comment (F-Kirchhoff) [inline src/app/components/Form/Form.tsx:12]: you can use the 'for' attribute of the label to link it to the input element

File: src/app/components/MalusBar/MalusBar.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './MalusBar.module.css';
+
+export default function MalusBar(): JSX.Element {
+  return (
+    <div>
+      <p>Overweight detected! - Ask Gamemaster for possible malus!</p>
+    </div>
+  );
+}
```
Comment (F-Kirchhoff) [inline src/app/components/MalusBar/MalusBar.tsx:10]: Why did you create an own component for this? Do you plan to use it more often?

**Pull request merged**